src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_time_field/step_time_field.html

```diff
@@ -2,173 +2,214 @@
-          class="kuiLabel"
-          for="timeFilterFieldSelect"
-        >
-          Time Filter field name
-        </label>
+  <p class="euiTextColor euiTextColor--subdued">
+    You've defined <strong>{{stepTimeField.indexPatternName}}</strong> as your index pattern. Now you can specify some settings before we create it.
+  </p>
 
-        <a
-          ng-if="!stepTimeField.isFetchingTimeFieldOptions"
-          class="kuiLink kuiSubText"
-          ng-click="stepTimeField.fetchTimeFieldOptions()"
-          kbn-accessible-click
-          aria-describedby="timeFilterFieldSelectLabel"
-        >
-          Refresh
-        </a>
+  <div class="euiSpacer euiSpacer--m"></div>
 
-        <p
-          ng-if="stepTimeField.isFetchingTimeFieldOptions"
-          class="kuiText kuiSubduedText"
-          aria-label="Please wait while we fetch your time field options"
-          aria-describedby="timeFilterFieldSelectLabel"
-        >
-          <span
-            aria-hidden="true"
-            class="kuiIcon fa-spinner fa-spin"
-          ></span>
-        </p>
-      </div>
+  <div class="euiFormRow">
+    <div class="euiFormLabel">
+      <div class="euiFlexGroup euiFlexGroup--justifyContentSpaceBetween">
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <label
+            id="timeFilterFieldSelectLabel"
+            for="timeFilterFieldSelect"
+          >
+            Time Filter field name
+          </label>
+        </div>
 
-      <div class="kuiVerticalRhythmSmall kuiVerticalRhythm">
-        <select
-          ng-show="stepTimeField.canShowMainSelect()"
-          id="timeFilterFieldSelect"
-          aria-describedby="timeFilterFieldSelectHelpText"
-          class="kuiSelect kuiSelect--large"
-          data-test-subj="createIndexPatternTimeFieldSelect"
-          ng-disabled="stepTimeField.isTimeFieldSelectDisabled()"
-          ng-required="stepTimeField.hasTimeFieldOptions()"
-          ng-options="option as option.display disable when option.isDisabled for option in stepTimeField.timeFieldOptions"
-          ng-model="stepTimeField.selectedTimeFieldOption"
-        ></select>
-        <select
-          ng-show="stepTimeField.canShowLoadingSelect()"
-          id="timeFilterFieldSelect"
-          aria-describedby="timeFilterFieldSelectHelpText"
-          class="kuiSelect kuiSelect--large"
-          data-test-subj="createIndexPatternTimeFieldSelect"
-          disabled="disabled"
-        >
-          <option>Loading...</option>
-        </select>
-        <p
-          ng-if="stepTimeField.canShowNoTimeBasedFieldsMessage()"
-          class="kuiText kuiSubduedText"
-        >
-          The indices which match this index pattern don't contain any time fields.
-        </p>
+        <div class="euiFlexItem euiFlexItem--flexGrowZero">
+          <a
+            ng-if="!stepTimeField.isFetchingTimeFieldOptions"
+            class="euiLink euiText euiText--extraSmall"
```

Comment:
Can we add this class here?

```html
class="euiLink euiText euiText--extraSmall timeFieldRefreshButton"
```

And then in `step_time_field.less` we can add the class:

```scss
/**
 * 1. Bring the line-height down or else this link expands its container when it becomes visible.
 */
.timeFieldRefreshButton {
  line-height: 1 !important; /* 1 */
}
```

There was a minor layout shift which occurs when you click the refresh button and this change will fix it.

Reply:
👍